symcrypt-sys/inc/static_LinuxDefault.c

```diff
@@ -0,0 +1,90 @@
+//
+// static_LinuxDefault.c
+// Default implementation for Linux static shared object.
+//
+// Copyright (c) Microsoft Corporation. Licensed under the MIT license.
+//
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "wrapper.h"
+#include <sys/random.h>
+#include <errno.h>
+
+SYMCRYPT_ENVIRONMENT_POSIX_USERMODE;
+
+VOID
+SYMCRYPT_CALL
+SymCryptModuleInit(

```

Comment:
Exposing ` SymCryptModuleInit` in the static link that under the covers calls `SymCryptInit` so that we don't have to add `SymCryptInit` to the bindings. 

The reasoning for this is that it exposes the `SymCryptInit` API to the public and will symbol undefined errors if called from a dynamic instance of `symcrypt-sys`